pkg/apis/core/validation/validation.go

```diff
@@ -2920,6 +2928,18 @@ func ValidateVolumeMounts(mounts []core.VolumeMount, voldevices map[string]strin
 			allErrs = append(allErrs, field.Invalid(idxPath.Child("mountPath"), mnt.MountPath, "must not already exist as a path in volumeDevices"))
 		}
 
+		// Disallow subPath/subPathExpr for image volumes
+		if utilfeature.DefaultFeatureGate.Enabled(features.ImageVolume) {
+			if v, ok := volumes[mnt.Name]; ok && v.Image != nil {
+				if mnt.SubPath != "" {
```

Comment:
kube standard is `len(mnt.SubPath) != 0`.  Please update here and below.

Reply:
Fixed as suggested.